packages/ui/src/council/components/election/CandidateVote/CandidateVote.tsx

```diff
@@ -76,7 +79,9 @@ export const CandidateVote = ({
           <StakeAndVotesRow>
             {hasOwnStake && (
               <>
-                <Subscription>My Stake</Subscription>
+                <Tooltip tooltipText={tooltipLabel}>
+                  <Subscription>My contributed votes</Subscription>
```

Comment:
Just to confirm, is this being used on both Reveal Vote screen and Past Elections?

Reply:
> Just to confirm, is this being used on both Reveal Vote screen and Past Elections?

yes. I guess so. I confirmed it on Past Elections screen in my end, and but couldn't check it "Reveal Vote" screen, because i couldn't be attended on Vote screen. but in the code level, I checked those 2 screens are using same components.

Reply:
ok, we can leave that to QA.